payments-core/src/main/java/com/stripe/android/CreateIntentCallback.kt

```diff
@@ -0,0 +1,32 @@
+package com.stripe.android
+
+import androidx.annotation.RestrictTo
+
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+fun interface CreateIntentCallback {
```

Comment:
There’s still some work to do around modelling this callback and their callback-based alternatives. We’ll do this in a separate pull request, but want to unblock work on the interceptor by including a minimal version of them here.